docs/content/configuration/creating_volumesnapshot.md

```diff
@@ -2,17 +2,17 @@
 
 Create a VolumeSnapshot YAML file for a specific PersistentVolumeClaim (PVC).
 
-VolumeSnapshotClass needs to be present before a VolumeSnapshot can be created. For more information, see [Creating a VolumeSnapshotClass](csi_ug_config_create_vol_snapshotclass.md).
+VolumeSnapshotClass needs to be present before a VolumeSnapshot can be created. For more information, see [Creating a VolumeSnapshotClass](creating_volumesnapshotclass.md).
 
 **Note:**
 
 -   IBM® FlashCopy® function is referred to as the more generic volume snapshots and cloning within this documentation set. Not all supported products use the FlashCopy function terminology.
 
-When creating volume snapshots, be sure to follow all of the snapshot configurations, found in [Compatibility and requirements](../installation/csi_ug_requirements.md) before snapshot creation.
+When creating volume snapshots, be sure to follow all of the snapshot configurations, found in [Compatibility and requirements](../installation/install_compatibility_requirements.md) before snapshot creation.
 
 1.  Create a snapshot for a specific PersistentVolumeClaim (PVC) using the `demo-volumesnapshot.yaml`.
 
-    For more information about PVC configuration, see [Creating a PersistentVolumeClaim (PVC)](csi_ug_config_create_pvc.md).
+    For more information about PVC configuration, see [Creating a PersistentVolumeClaim (PVC)](creating_pvc.md).
 
     ```
     apiVersion: snapshot.storage.k8s.io/v1beta1
```

Comment:
@ArbelNathan do we have a ticket to move our examples to `snapshot.storage.k8s.io/v1`?

Reply:
nevermind, I see we have CSI-2873